pkg/scale/scale.go

```diff
@@ -57,8 +57,10 @@ func (s *Scaler) EnsureScale(ctx context.Context, scale uint, gRPCPort int32) er
 	// make use of reclaim policy = delete. A reclaim policy of retain is fine
 	// but will result in wasted money, recycle should be considered unsafe and
 	// is officially deprecated by kubernetes.
-	if err := s.PVCPruner.Prune(ctx); err != nil {
-		return errors.Wrap(err, "initial PVC pruning")
+	if usePVCPruner {
+		if err := s.PVCPruner.Prune(ctx); err != nil {
+			return errors.Wrap(err, "initial PVC pruning")
+		}
```

Comment:
Put an else here logging that we are not removing the pvc

Reply:
done